flask_resty/testing.py

```diff
@@ -97,16 +121,7 @@ def assert_response(response, expected_status_code, expected_data=UNDEFINED):
     else:
         response_data = get_errors(response)
 
-    assert_shape(response_data, expected_data)
-
+    if not isinstance(expected_data, Predicate):
```

Comment:
seems like the opposite, this affects all current use cases, since expected_data is NOT usually a predicate, then will be wrapped in a Shape. doing `assert_shape` on a Shape is pointless, isn't it?